tests/test_schemas.py

```diff
@@ -14,6 +14,7 @@
 
 import tests
 import ga4gh.protocol as protocol
+import tests.utils
 
 # TODO We should add some tests here - see comment from @pashields on
 # https://github.com/ga4gh/server/pull/39
```

Comment:
I think there is more to do here, but I'm not sure how valuable this comment is at this point. What say you @jeromekelleher ?